app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1><%= link_to "TaskList", tasks_path %></h1>
+<h2>just get it done!</h2>
+
+<body>
+    <section class="task_list">
+        <div>
+            <p class="button"><%= link_to "Add Task", new_task_path %></p>
+        </div>
+        <div class="task_list">
+            <p>ToDo:</p>
+            <ol>
+                <% @tasks.each do |task| %>
+                    <li>
+                        <%if task.complete == false%>
+                            <div>
+                                <a href="/tasks/<%=task.id%>"> <%= task.name %>: <%= task.description %> Due:<%= task.completion_date %> </a>
```

Comment:
Best pracitce would be to use the `link_to` helper, and also to reference the path with the path helper (here, `task_path(task)` instead of `/tasks/...`